### ISSUE TITLE
Implementing a Feature Request

### ISSUE THREAD

**Issue by Jopchumba** [Devs-Garden/jsonbase#9]
@syamdanda thanks for creating the awesome project that will help speed up testing projects that depend on databases.
are you looking into creating a CLI tool for the project to make it more interactive 
thanks looking forward to hearing from you.

**Comment by syamdanda** [comment]
Yes @Jopchumba, this is just a pre-alpha release. Yet to come more features..

**Comment by Jopchumba** [comment]
@syamdanda I see  it's a promising tool I like the way it is maybe we can collaborate and find more solutions

**Comment by syamdanda** [comment]
good thought @Jopchumba, looking forward to include that feature.
